### ISSUE TITLE
Support installation as a Windows service

### ISSUE THREAD

**Issue by jirihajek** [mediamonkeyserver/mms#8]
Should be quite easy using https://www.npmjs.com/package/node-windows. Must be tested, how it works with 'pkg' though.

When this is implemented, we should also create a proper installer, so that the service can be properly managed (installed/uninstalled/updated).

**Comment by JL102** [comment]
Yup. I have my Node-Express-Pug-based website, drakinite.net, running on my home server using node-windows. It successfully runs on boot without even having to log in to the server. (Unfortunately I'm having network issues with my ISP, and I'm not home at the moment, so you won't be able to load the webpage - but trust me, it does run, and it is consistent & dependable. :P)

https://github.com/JL102/drakinite.net/blob/master/install-service.js If I recall correctly, all I had to do was run `node install-service.js` with this short script I made, and it worked fine.  The installation process is a little strange, because it requests elevation 3-4 times in a row, but it worked. Maybe if we initially elevate the node process it won't need to request multiple times in a row. I don't remember if I had to manually enable the service on boot or not, but I'm sure that step can be done by the installer whenever it's done.

Also, I made it so that my debug logs are written to a file, which may be useful to implement for debugging.
